hamocc/mo_hamocc4bcm.F90

```diff
@@ -75,36 +71,36 @@ subroutine hamocc4bcm(kpie,kpje,kpke,kbnd,kplyear,kplmon,kplday,kldtday,&
-    real,    intent(in)    :: pfu10  (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! absolute wind speed at 10m height [m/s]
-    real,    intent(in)    :: ptho   (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd,kpke) ! potential temperature [deg C].
-    real,    intent(in)    :: psao   (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd,kpke) ! salinity [psu.].
-    real,    intent(in)    :: patmco2(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! atmospheric CO2 concentration [ppm] used in fully coupled mode
-    real,    intent(out)   :: pflxco2(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! CO2 flux [kg/m^2/s].
-    real,    intent(inout) :: pflxdms(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! DMS flux [kg/m^2/s].
-    real,    intent(in)    :: patmbromo(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)    ! atmospheric bromoform concentration [ppt] used in fully coupled mode.
-    real,    intent(inout) :: pflxbromo(1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)    ! Bromoform flux [kg/m^2/s].
+    integer, intent(in)  :: kpie                                            ! 1st dimension of model grid.
+    integer, intent(in)  :: kpje                                            ! 2nd dimension of model grid.
+    integer, intent(in)  :: kpke                                            ! 3rd (vertical) dimension of model grid.
+    integer, intent(in)  :: kbnd                                            ! number of halo grid points.
+    integer, intent(in)  :: kplyear                                         ! current year.
+    integer, intent(in)  :: kplmon                                          ! current month.
+    integer, intent(in)  :: kplday                                          ! current day.
+    integer, intent(in)  :: kldtday                                         ! number of time step in current day.
+    real,    intent(in)  :: pdlxp  (kpie,kpje)                              ! size of grid cell (longitudinal) [m].
+    real,    intent(in)  :: pdlyp  (kpie,kpje)                              ! size of grid cell (latitudinal) [m].
+    real,    intent(in)  :: pddpo  (kpie,kpje,kpke)                         ! size of grid cell (depth) [m].
+    real,    intent(in)  :: prho   (kpie,kpje,kpke)                         ! density [g/cm^3].
+    real,    intent(in)  :: pglat  (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! latitude of grid cells [deg north].
+    real,    intent(in)  :: omask  (kpie,kpje)                              ! land/ocean mask.
+    real,    intent(in)  :: dust   (kpie,kpje)                              ! dust deposition flux [kg/m2/month].
+    real,    intent(in)  :: rivin  (kpie,kpje,nriv)                         ! riverine input [kmol m-2 yr-1].
+    real,    intent(in)  :: ndep   (kpie,kpje)                              ! nitrogen deposition [kmol m-2 yr-1].
+    real,    intent(in)  :: oafx   (kpie,kpje)                              ! alkalinity flux from alkalinization [kmol m-2 yr-1]
+    real,    intent(in)  :: pi_ph  (kpie,kpje)                              
```

Comment:
Hi Jörg, can you add a comment, what `pi_ph` is?

Reply:
done